src/main/java/org/commonwl/view/cwl/CWLToolRunner.java

```diff
@@ -171,6 +171,7 @@ public void createWorkflowFromQueued(QueuedWorkflow queuedWorkflow)
       FileUtils.deleteGitRepository(repo);
     } finally {
       gitSemaphore.release(repoUrl);
+      FileUtils.deleteTemporaryGitRepository(repo);
```

Comment:
Hmm.. no test coverage here
https://coveralls.io/builds/55575771/source?filename=src%2Fmain%2Fjava%2Forg%2Fcommonwl%2Fview%2Fcwl%2FCWLToolRunner.java#L174

Reply:
Testing this line requires to simulate concurrency. Good luck :)